src/game.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Board from './board';
+
+export default class Game extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            history: [{
+                squares: Array(9).fill(null),
+            }],
+            currentMove: 0,
+            xIsNext: true,
+        };
+    }
+
+    handleClick(i) {
+        const history = this.state.history.slice(0, this.state.currentMove + 1);
+        const current = history[this.state.currentMove];
+        const squares = current.squares.slice();
+        if (calculateWinner(squares) || squares[i]) {
+            return;
+        }
+        squares[i] = this.state.xIsNext ? 'X' : 'O';
+        this.setState({
+            history: history.concat([{ squares }]),
+            currentMove: this.state.currentMove + 1,
+            xIsNext: !this.state.xIsNext,
+        });
+    }
+
+    jumpTo(i) {
+        this.setState({
+            currentMove: i,
+            xIsNext: i % 2 === 0,
+        });
+    }
+
+    render() {
+        const history = this.state.history;
```

Comment:
minor, could use exciting es6 destructuring :)

const { history } = this.state;